app/models/language.rb

```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+
 class Language < ActiveRecord::Base
-  validates_presence_of :code, :name
+  validates :code, presence: true
+  validates :name, presence: true
+
```

Comment:
you can combine these since they are validating same thing.

Reply:
👍